### PR TITLE
Add dutch translation

### PR DESCRIPTION
**Pull request by koenpunt**

### CODE REVIEW

File: nl.js.coffee [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    table_save: "Opslaan"
+    table_cancel: "Annuleren"
+    table_delete: "Tabel Verwijderen"
+    table_clone: "Tabel Dupliceren"
+    table_add_field: "Veld toevoegen"
+    field_name: "Naam"
+    field_type: "Type"
+    field_size: "Grootte"
+    field_default: "Staandaardwaarde"
+    field_allow_nulls: """"nulls" toestaan"""
```
Comment (felladrin) [inline nl.js.coffee]: I guess this line has more quotation marks than it needed. Is this correct?

Comment (koenpunt) [inline nl.js.coffee]: Triple quotes are necessary when using the same quotes in the string again. It is being used in the English translation as well ;)

Comment (felladrin) [inline nl.js.coffee]: Oh! Alright then! :)

**Comment by cenan** [comment]
@koenpunt thank you for the translation :)

**Pull request merged**